types/npmlog/npmlog-tests.ts

```diff
@@ -12,7 +12,8 @@ npmlog.http(prefix, message);
 npmlog.warn(prefix, message);
 npmlog.error(prefix, message);
 
-npmlog.level = "silly";
+// ES6 module does't support changing variable exported by `export let`
```

Comment:
You can write an import of the style `import npmlog = require("npmlog")` if you want to test this in the future.